hack/utils.sh

```diff
@@ -17,9 +17,9 @@
 ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"/..
 
 # Not from vendor.conf.
-CRITOOL_VERSION=v1.18.0
+CRITOOL_VERSION=selinux
 CRITOOL_PKG=github.com/kubernetes-sigs/cri-tools
-CRITOOL_REPO=github.com/kubernetes-sigs/cri-tools
+CRITOOL_REPO=github.com/erikwilson/cri-tools
```

Comment:
Just making a comment as a note, we will have to update this back to upstream once all the upstream PRs are merged into cri-tools

Reply:
The initial PR for MCS testing has been merged and is now available at https://github.com/kubernetes-sigs/cri-tools/commit/89384cc13a27bb9128553c9fe75a7cc07c6a95bb

Reply:
Thanks, i'll update back to upstream